### DISCUSSION TITLE
Modify branch name in commit dialog

### DISCUSSION THREAD

**Discussion by jgrumboe** [community/community#110855] [Mobile]
Hi,

It would be great to modify the branch name in the commit dialog after editing a file in GitHub mobile. So far we can either select to commit to main or a new branch with fixed name <user>-patch-<number>. I'd like to enter my own branch name or modify the suggested one.

Thanks, Jo

**Comment by alcere** [comment]
Hi @jgrumboe ! Thank you for your feedback! I have opened an internal issue and the team will evaluate your proposal to improve the commit flow in the future. Thank you for using GitHub mobile!
